plugin/metrics/prometheus/options.go

```diff
@@ -48,14 +48,10 @@ const (
 	defaultNormalizeDuration           = false
 )
 
-type namespaceConfig struct {
-	config.Configuration `mapstructure:",squash"`
-	namespace            string
-}
-
 // Options stores the configuration entries for this storage.
 type Options struct {
-	Primary namespaceConfig `mapstructure:",squash"`
+	config.Configuration `mapstructure:",squash"`
+	namespace            string
```

Comment:
I don't believe we need this. Is it ever non-blank in any of the mains?

Reply:
https://github.com/jaegertracing/jaeger/blob/e086ce1bead93371cef4d80325878de0dee681e9/plugin/metrics/prometheus/factory.go#L43

This was the only non-blank instance that I could find. We could do without this.

Reply:
Yeah, that should just be the default prefix, we don't need it to be customizable. I would declare it as a constant and use it instead of `opt.namespace+`

Reply:
Okay.